evalml/pipelines/utils.py

```diff
@@ -177,8 +177,10 @@ def generate_pipeline_code(element):
     code_strings = ['import json']
     if not isinstance(element, PipelineBase):
         raise ValueError("Element must be a pipeline instance, received {}".format(type(element)))
+    if isinstance(element.component_graph, dict):
+        raise ValueError("Code generation for nonlinear pipelines is not supported yet")
```

Comment:
I'm not sure how much work would be required for implementing code generation for nonlinear pipelines, is it worth it to file a separate issue to track this?

Reply:
I think it should be a separate issue!